kc-app/src/App.js

```diff
@@ -57,6 +57,8 @@ function App() {
                 setTab('identity');
             }
             else {
+                const registries = await keymaster.listRegistries();
+                setRegistries(registries);
```

Comment:
Rather than duplicating the code in the else, the registries should be set after the if/else